functions.php

```diff
@@ -4,7 +4,7 @@
  *
  * @package Memberlite
  */
-define( 'MEMBERLITE_VERSION', '4.5.3' );
+define( 'MEMBERLITE_VERSION', '4.5.4.1.91' );
```

Comment:
> I bump the functions only to force break css cache. Had to do that since our sites are running the grid layout branch

I'm going to resolve this as unneeded